kyuubi-metrics/pom.xml

```diff
@@ -88,6 +88,11 @@
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Does the guava package need to be shaded?

Reply:
it is only used at Kyuubi Server, so there is no other version of guava

Reply:
> it is only used at Kyuubi Server, so there is no other version of guava

@ulysses-you, no matter whether the guava package is only used at Kyuubi Server, the guava package is better to be shaded for the users.

Reply:
I'm afraid we not do that, because curator need it https://cwiki.apache.org/confluence/display/CURATOR/TN13

Reply:
Thanks, @SteNicholas  I checked the guava usages in this module, I think we can remove guava here